friture/pitch_tracker.py

```diff
@@ -102,17 +107,37 @@ def __init__(self, parent, engine):
 
         self.gridLayout.addWidget(self.quickWidget, 0, 0, 1, 1)
 
+        self.pitch_view_model = PitchViewModel(self)
```

Comment:
In practice, there is already a viewModel here, it's Scope_Data above. Could Scope_Data be used as a based for a new class that adds the pitch properties, like Spectrum_Data extends Scope_Data ?